requirements.txt

```diff
@@ -1 +1,4 @@
-cocotb
\ No newline at end of file
+cocotb==1.8.0
+cocotb-test==0.2.4
+pytest==7.4.0
+pyyaml
```

Comment:
Why do we need pyyaml?

Reply:
@JosseVanDelm No worries! You know these stuff more than I do so I'm learning a lot :D sorry if I'm too noob hahaha

Also, the pyyaml is needed for the yml source files that Francesco used. He already listed the paths there so I figured not to create new filelists but use his files instead since they're already available.

Merging!